### PR TITLE
i18n(zh-cn): translate `guides/prefetch.mdx` and update `guides/integrations-guide/prefetch.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### Description (required)

- translate `guides/prefetch.mdx` to Chinese version.
- update `guides/integrations-guide/prefetch.mdx`.

Note that **this PR should be merged after #5349** 

#### Related issues & labels (optional)

#5324

### CODE REVIEW

File: src/content/docs/zh-cn/guides/prefetch.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+你可以通过将其传递给 `data-astro-prefetch` 属性来为单个链接指定策略：
+
+```html
+<a href="/about" data-astro-prefetch="tap">About</a>
+```
+
+每种策略都经过了微调，只有在需要时才预获取，以节省用户的带宽。例如：
+
+- 如果访问者使用 [数据保护模式](https://developer.mozilla.org/zh-CN/docs/Web/API/NetworkInformation/saveData) 或者有 [慢速连接](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType)，则预获取将被关闭。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/prefetch.mdx:49]:
```suggestion
- 如果访问者使用[流量节省模式](https://developer.mozilla.org/zh-CN/docs/Web/API/NetworkInformation/saveData)或者有[慢速连接](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType)，则预获取将被关闭。
```

Reference: https://source.android.com/docs/core/data/data-saver?hl=zh-cn

File: src/content/docs/zh-cn/guides/prefetch.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+```js
+// 使用 `fetch()` 预获取，它具有更高的优先级。
+prefetch('/about', { with: 'fetch' });
+
+// 使用 `<link rel="prefetch">` 预获取，它具有较低的优先级
+// 并由浏览器手动调度。（默认）
+prefetch('/about', { with: 'link' });
+```
+
+`prefetch()` API 包括相同的 [数据保护模式](https://developer.mozilla.org/zh-CN/docs/Web/API/NetworkInformation/saveData) 和 [慢速连接](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType) 检测，因此只有在需要时才预获取。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/prefetch.mdx:117]:
```suggestion
`prefetch()` API 包括相同的[流量节省模式](https://developer.mozilla.org/zh-CN/docs/Web/API/NetworkInformation/saveData)和[慢速连接](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType)检测，因此只有在需要时才预获取。
```

**Review by Genteure — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**